std_msgs/Header header
erau_fusion_msgs/Tracked_Lane[32] objects
